src/net_processing.h

```diff
@@ -31,6 +31,7 @@ struct CNodeStateStats {
     std::vector<int> vHeightInFlight;
     uint64_t m_addr_processed = 0;
     uint64_t m_addr_rate_limited = 0;
+    bool m_addr_relay_enabled;
```

Comment:
Maybe explicitly default initialize this to `false`:

```suggestion
    bool m_addr_relay_enabled{false};
```

Reply:
done